src/js/angular/ttyg/templates/chat-view.html

```diff
@@ -0,0 +1,28 @@
+<link href="css/ttyg/chat-view.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="chat-view">
+    <div class="messages" ng-class="{'hidden': !selectedChat}">
+        <div id="messages-scrollable">
+            <div class="messages-hint text-muted" ng-if="!selectedChat.messages.length">
+                {{ 'ttyg.hint' | translate }}
+            </div>
+            <div ng-repeat="question in selectedChat.messages" class="m-1"
+                 ng-class="{'mt-3': question.role === 'user'}">
+                <span ng-if="question.role === 'user'">User:  </span>
+                <span class="messages-item-content">{{ question.message }}</span>
+            </div>
+            <div onto-loader-new ng-show="loader" size="40"></div>
+        </div>
+    </div>
+    <div class="question">
+        <form>
+            <div class="mb-2 d-flex">
+                <input class="form-control" ng-model="chatQuestion.question" type="text">
```

Comment:
Isn't it better the model to be `chatQuestion.message` and not `chatQuestion.question`. It's kind of awkward this way. I think we should revise the other models as well, because I think we have other similar inconsistencies.

Reply:
Yes, I agree with you. I had planned to make changes to the models as we discussed, but the idea of this merge request is to reorganize the code a little bit, not to implement all changes at once, as that would make it harder to review.

Reply:
but this is trivial stuff and is related with this feature